plugins/techdocs-backend/src/helpers.ts

```diff
@@ -146,10 +146,18 @@ export const checkoutGitRepository = async (
 
   if (token) {
     const type = getGitRepoType(repoUrl);
-    const auth = type === 'github' ? `${token}:x-oauth-basic` : `:${token}`;
-    parsedGitLocation.token = auth;
+    switch (type) {
+      case 'gitlab':
+        // Personal Access Token
+        parsedGitLocation.token = `dummyUsername:${token}`;
+        break;
+      case 'github':
+      default:
```

Comment:
Shouldn't `default` be `:${token}` to be the same as before for the other VCS hosts?

Reply:
@adrianke77 Agreed - this changes the default. I think the logic should be

if github - `${token}:x-oauth-basic`
if gitlab - `dummyUsername:${token}`
else - `:${token}`

Reply:
Updated per comments

Reply:
Thanks @adrianke77 ! Will review/test this today.